src/utils/formatFilter.js

```diff
@@ -0,0 +1,24 @@
+function formatFilter (modelName, search, filters) {
+  const model = require('./models').getModel(modelName)
+  const searchLikeTypes = ['text', 'textarea']
+
+  const formattedFilters = []
+
+  const { Like } = require('typeorm')
```

Comment:
Qual o critério que está utilizando para definir a ordem de variáveis e _requires_?

Reply:
nesse caso nenhum pq n importa a ordem

Reply:
É interessante pensar em alguma organização, se o volume for maior pode te atrapalhar.

Reply:
Sim, mas nesse caso não tem mt o pq, normalmente vou criando as variaveis de acordo com as necessidades e colocando perto das que tem contexto com ela, mas nesse caso todos fazem parte do msm contexto, oq da pra fazer ai é tirar os espaços talvez e deixar os require juntos